scripts/setup/configure-rabbitmq

```diff
@@ -5,11 +5,17 @@
 set -e
 set -x
 
+RABBITMQ_ADMIN="${RABBITMQ_ADMIN:-rabbit}"
```

Comment:
So, looking into how RabbitMQ works more, nodenames are by default `rabbit@hostname`, and the rabbit in there has nothing to do with a user or admin or anything -- it's just a name.  So I think we should get rid of the `RABBITMQ_ADMIN` variable and instead just have a single variable `RABBITMQ_NODENAME` which defaults to `rabbit@hostname`.